.github/workflows/test.yml

```diff
@@ -0,0 +1,80 @@
+name: Tests
+
+env:
+  CONDA_STORE_SERVER_PORT: 8080
+  CONDA_STORE_BASE_URL: http://localhost:{{ $CONDA_STORE_SERVER_PORT }}
+  CONDA_STORE_AUTH: basic
+  CONDA_STORE_USERNAME: username
+  CONDA_STORE_PASSWORD: password
+  REACT_APP_API_URL: http://localhost:5000/conda-store/
+  REACT_APP_AUTH_METHOD: cookie
+  REACT_APP_LOGIN_PAGE_URL: http://localhost:5000/conda-store/login?next=
+  REACT_APP_AUTH_TOKEN: 
+  REACT_APP_STYLE_TYPE: green-accent
+  REACT_APP_CONTEXT: webapp
+  REACT_APP_SHOW_AUTH_BUTTON: true
+  REACT_APP_LOGOUT_PAGE_URL: http://localhost:5000/conda-store/logout?next=/ 
+  YARN_ENABLE_IMMUTABLE_INSTALLS: false
```

Comment:
This env details don't match with the `.env.example` file

Reply:
fixed